WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterFragment.kt

```diff
@@ -99,9 +103,19 @@ class ActivityLogTypeFilterFragment : DialogFragment() {
         viewModel.dismissDialog.observe(viewLifecycleOwner, Observer {
             it.applyIfNotHandled { dismiss() }
         })
+
+        val dateRange: DateRange? = if (requireNotNull(arguments).containsKey(ARG_DATE_RANGE_AFTER)) {
```

Comment:
Question (❓): What if we make this more concrete by checking the `ARG_DATE_RANGE_BEFORE` as well? I understand that the `ARG_DATE_RANGE_AFTER` is assuming that the `ARG_DATE_RANGE_BEFORE` is available, but what if a developer in the future makes a change that somehow alters this assumption? Is it better to make this more concrete now that have it potentially fail in the future. Wdyt?

PS: Also, maybe by having that more concrete improves the intention readability, don't you think?

Reply:
I wanted to follow the fail fast principle. However, I agree that the code isn't easy to follow. I made some changes in 94c937b344b326087bac64989631f4d7e6158b6a, let me know what you think.

Reply:
@malinajirka I can't find the changes following this hash, can you maybe link it?

Reply:
I usually reply to comments as I go through the review and push the changes in the end. In general, it's better to wait for my "request re-review" request before you as reviewer start replying. Either way, F5/cmd+R, should solve this issue 
 now ;).

Reply:
Thank YOU! Yeap, I was trying to be fast, next time I'll just wait... 😅

Reply:
Thank you ;)!

I wish GitHub had the "code review (submit at the end)" mode even when working on the PR feedback.